### PR TITLE
More detail for Do Not Rewrite syntax

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/safe-links.md [hunk 1]
```diff
@@ -300,12 +300,16 @@ To add entries to the list in new or existing Safe Links policies, see [Create S
   - Microsoft Teams
   - Office web apps
 
-  For a truly universal list of URLs that are allowed everywhere, see [Manage the Tenant Allow/Block List](tenant-allow-block-list.md).
+  For a truly universal list of URLs that are allowed everywhere, see [Manage the Tenant Allow/Block List](tenant-allow-block-list.md).  However, note that URLs added there will not be excluded from Safe Links rewriting, as that must be done in a Safe Links policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-links.md]:
```suggestion
  For a truly universal list of URLs that are allowed everywhere, see [Manage the Tenant Allow/Block List](tenant-allow-block-list.md). However, note that URLs added there will not be excluded from Safe Links rewriting, as that must be done in a Safe Links policy.
```

File: microsoft-365/security/office-365-security/safe-links.md [hunk 2]
```diff
@@ -300,12 +300,16 @@ To add entries to the list in new or existing Safe Links policies, see [Create S
...
 - If you already have **Do not rewrite the following URLs** entries in your Safe Links policies, be sure to review the lists and add wildcards as required. For example, your list has an entry like `https://contoso.com/a` and you later decide to include subpaths like `https://contoso.com/a/b`. Instead of adding a new entry, add a wildcard to the existing entry so it becomes `https://contoso.com/a/*`.
 - You can include up to three wildcards (`*`) per URL entry. Wildcards explicitly include prefixes or subdomains. For example, the entry `contoso.com` is not the same as `*.contoso.com/*`, because `*.contoso.com/*` allows people to visit subdomains and paths in the specified domain.
 - If a URL uses automatic redirection for HTTP to HTTPS (for example, 302 redirection for `http://www.contoso.com` to `https://www.contoso.com`), and you try to enter both HTTP and HTTPS entries for the same URL to the list, you might notice that the second URL entry replaces the first URL entry. This behavior does not occur if the HTTP and HTTPS versions of the URL are completely separate.
+- Do not specify http:// or https:// (i.e., contoso.com) in order to exclude both http and https versions.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-links.md]:
```suggestion
- Do not specify http:// or https:// (that is, contoso.com) in order to exclude both HTTP and HTTPS versions.
```

File: microsoft-365/security/office-365-security/safe-links.md [hunk 3]
```diff
@@ -300,12 +300,16 @@ To add entries to the list in new or existing Safe Links policies, see [Create S
...
+- `*.contoso.com` does NOT cover contoso.com, so you'd need to exclude both to cover both the specified domain and any child domains.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-links.md]:
```suggestion
- `*.contoso.com` does **not** cover contoso.com, so you would need to exclude both to cover both the specified domain and any child domains.
```

File: microsoft-365/security/office-365-security/safe-links.md [hunk 4]
```diff
@@ -300,12 +300,16 @@ To add entries to the list in new or existing Safe Links policies, see [Create S
...
+- `contoso.com/*` DOES cover just contoso.com, so there's no need to exclude both `contoso.com` and `contoso.com/*`, rather just `contoso.com/*` would suffice.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-links.md]:
```suggestion
- `contoso.com/*` covers **only** contoso.com, so there's no need to exclude both `contoso.com` and `contoso.com/*`; just `contoso.com/*` would suffice.
```

File: microsoft-365/security/office-365-security/safe-links.md [hunk 5]
```diff
@@ -300,12 +300,16 @@ To add entries to the list in new or existing Safe Links policies, see [Create S
...
+- To exclude all iterations of a domain, it takes two exclusion entries:  `contoso.com/*` and `*.contoso.com/*` (these combine to exclude both http/https, the main domain contoso.com and/or any child domains, as well as any or not ending part (e.g., contoso.com and contoso.com/vdir1 are covered).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-links.md]:
```suggestion
- To exclude all iterations of a domain, two exclusion entries are needed; `contoso.com/*` and `*.contoso.com/*`. These combine to exclude both HTTP and HTTPS, the main domain contoso.com and any child domains, as well as any or not ending part (for example, both contoso.com and contoso.com/vdir1 are covered).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**